dev-tools/src/utils/logger.ts

```diff
@@ -0,0 +1,21 @@
+import { Logger } from '@aws-lambda-powertools/logger'
+import { Context } from 'aws-lambda'
+
+const loggerInstance = new Logger({
+  serviceName: process.env.AWS_LAMBDA_FUNCTION_NAME,
+  logLevel: process.env.LOG_LEVEL || 'DEBUG',
+  environment: process.env.ENVIRONMENT
+})
+
+export const initialiseLogger = (context: Context) => {
+  loggerInstance.addContext(context)
+  loggerInstance.removeKeys(['zendeskId'])
+}
+
+export const appendZendeskIdToLogger = (zendeskId: string) => {
+  if (typeof zendeskId !== 'undefined') {
```

Comment:
I think you make this simpler and just put `if (zendeskId)` because all you care about is if the variable has actually been set to something. As it is, somebody could pass in `null` and it would end up being appended to the logger.